.github/workflows/jsr-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish to JSR after npm publish
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main, beta, next]
+  workflow_dispatch:
+
+jobs:
+  publish-jsr:
+    if: >-
+      github.event_name == 'workflow_dispatch' || 
+      (github.event_name == 'pull_request' && 
+       github.event.pull_request.merged == true && 
+       contains(github.event.pull_request.title, 'chore: version packages'))
```

Comment:
Added CI to trigger `jsr publish` upon merging the changeset PR (`chore: version packages`), but this hasn't been tested yet.